### PR TITLE
urlmatcher: Add page description

### PR DESCRIPTION
**Pull request by mrshu**
- Added a way of finding the page description and using it instead of
  the readable summary which is many times wrong.

Signed-off-by: mr.Shu mr@shu.io

### CODE REVIEW

**Comment by mrshu** [comment]
to @adman @pepol
cc @jakubnvk

**Comment by mrshu** [comment]
ping @Adman @pepol

File: err-urlmatcher/urlmatcher.py [hunk 1]
```diff
@@ -44,14 +51,27 @@ def url_matcher(self, msg, match):
...
+
+        readable_description = self.text_cleanup(readable_description)
+
+        description = ''
+        if len(readable_description) > len(readable_article):
+            description = readable_description
+        else:
+            description = readable_article
+
+        return "{} > {} > {}".format(url,
```
Comment (mrshu) [inline err-urlmatcher/urlmatcher.py]: It might be interesting to use newlines here. Thoughts @pepol @Adman ?

Comment (pepol) [inline err-urlmatcher/urlmatcher.py]: Wouldn't that flood the channel too much?

Comment (mrshu) [inline err-urlmatcher/urlmatcher.py]: @pepol well, 3 lines is not that much I would say.

I would personaly like to give it a go but your question is very valid.

Comment (pepol) [inline err-urlmatcher/urlmatcher.py]: @mrshu before we merge this, can we try running canary with the newlines for couple of days, just to see how it works?

**Pull request merged**